### ISSUE TITLE
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Map.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'

### ISSUE THREAD

**Issue by deyanp** [Zaid-Ajaj/Feliz#318]
While experimenting for the first time with Feliz I added fsproj reference

`<PackageReference Include="Feliz.UseDeferred" Version="1.4.0" />`

and tried to create the sample component

```
[<ReactComponent>]
let CounterFromServer() =
    let data = React.useDeferred(Service.callApi, [||])

    match data with
    | Deferred.HasNotStartedYet -> Html.none
    | Deferred.InProgress -> Html.h1 "Loading"
    | Deferred.Failed error -> Html.span error.Message
    | Deferred.Resolved counter -> Html.h1 counter
```

however I am getting the errors

```
ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../Feliz/React.fs.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 32:0-296 180:21-63 184:12-32 185:14-34 186:2-34 187:11-42 209:2-30 216:10-52 216:54-74 216:76-96 216:98-130 216:132-160 222:26-46 243:2-34 244:11-42 248:14-46 257:10-30 257:32-64 257:66-98 263:21-51 266:16-48 274:12-32 275:14-34 276:2-34 277:11-42 308:14-41 315:2-30 324:10-42 324:44-64 324:66-86 324:88-120 324:122-149 324:151-179
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Async.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 33:0-171 184:33-56 188:6-12 210:4-18 222:47-70 245:6-12 249:9-32 250:6-20 274:33-56 278:6-12 311:6-20 311:30-38 311:73-79
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/AsyncBuilder.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 34:0-97 193:21-30 225:23-32 289:43-52
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/List.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 37:0-85 284:11-16
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Map.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 36:0-97 263:52-57 270:13-16 320:9-15
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Option.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 35:0-86 204:18-22 236:20-24 299:24-28
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Reflection.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 31:0-108 70:9-19 71:48-58
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Seq.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 38:0-119 284:17-22 285:13-20 289:15-26
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js

ERROR in ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js
Module not found: Error: Can't resolve '../docs/.fable/fable-library.3.0.0-nagareyama-rc-007/Types.js' in '/home/xxxxx/git/Experiments/FelizTests/FelizApp/src/.fable/Feliz.UseDeferred.1.4.0'
 @ ./src/.fable/Feliz.UseDeferred.1.4.0/UseDeferred.fs.js 30:0-86 68:2-7
 @ ./src/App.fs.js
 @ ./src/Main.fs.js
 @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/Main.fs.js
```

Any idea why?

`dotnet fable --help
`
gives

`Fable: F# to JS compiler 3.0.1`

...

**Comment by deyanp** [comment]
hmm, 

`npm i `

solved this ... but wondering if Feliz.UseDeferred is really still using fable-library.3.0.0-nagareyama-rc-007 ...

**Comment by Zaid-Ajaj** [comment]
Hi there @deyanp, I believe this is an issue because Feliz.UseDeferred accidentally included the generated JS files from the time I was testing it. I will publish a new version that doesn't include the JS files so that they are generated based on your recent Fable version

**Comment by Zaid-Ajaj** [comment]
Just published a batch of Feliz libs with the fix, can you please update to latest 1.4.0 -> 1.4.1? consider deleting `.fable` directory while you are at it to make sure the old artifacts are removed :smile: 

Please reopen the issue if the problem persists :smile:
